shell/common/shell.h

```diff
@@ -98,6 +98,20 @@ class Shell final : public PlatformView::Delegate,
  public:
   template <class T>
   using CreateCallback = std::function<std::unique_ptr<T>(Shell&)>;
+  typedef std::function<std::unique_ptr<Engine>(
```

Comment:
Kinda does the same thing as the above. Can we make the names sound similar like EngineCreateCallback or something?

Reply:
done